process/tasks/ABProcessElementCore.js

```diff
@@ -455,10 +455,10 @@ module.exports = class ABProcessTaskCore extends ABMLClass {
    }
 
    get objectOfStartElement() {
-      let startElem = this.startElements[0];
+      const startElem = this.startElements[0];
       if (!startElem) return null;
 
-      let startElemObj = this.application.objects(
+      const startElemObj = this.AB.objects(
```

Comment:
I think you can use `this.AB.objectByID(startElem.objectID)`

Reply:
Thank you. It should be updated.